docs/usage/configuration-options.md

```diff
@@ -1102,6 +1102,19 @@ If you allow Renovate to run on "Selected" repositories, `forkProcessing` will b
 Only the `onboardingConfigFileName` (which defaults to `renovate.json`) is supported for `forkProcessing`.
 You can't use other filenames because Renovate only checks the default filename when using the Git-hosting platform's API.
 
+## gerritLabelMapping
+
+The name of the Gerrit labels to use for the `StabilityDays` and `Merge-Confidence` status.
+
+```json
+{
+  "stabilityDaysLabel": "Renovate-Stability",
+  "mergeConfidenceLabel": "Renovate-Confidence"
+}
+```
```

Comment:
You probably forgot to update this part. 😉 

We renamed `stabilityDays` to `minimumReleaseAge`.

Reply:
sorry